ci/generate_azure_pipelines_matrices.py

```diff
@@ -217,10 +217,17 @@ def __init__(self, platform, build_jobs):
             subarchives="qtbase qttools qt icu",
         ),
         BuildJob(
+            # Archives stored as .7z
             "install-src", "6.1.0", "linux", "desktop", "gcc_64", "gcc_64", subarchives="qtlottie",
             # Fail the job if this path does not exist:
             check_output_cmd="ls -lh ./6.1.0/Src/qtlottie/",
         ),
+        BuildJob(
+            # Archives stored as .tar.gz
+            "install-src", "6.4.3", "linux", "desktop", "gcc_64", "gcc_64", subarchives="qtlottie",
+            # Fail the job if this path does not exist:
+            check_output_cmd="ls -lh ./6.4.3/Src/qtlottie/",
+        ),
```

Comment:
Not sure if the new `install-src` job should just replace the old one, or stay alongside it